corehq/apps/ivr/migrations/0002_call_app_id.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.22 on 2019-10-28 20:52
+from __future__ import unicode_literals
```

Comment:
you can drop the `__future__` and `utf-8` statements

Reply:
5dd2681db42

Reply:
Moved this change over to https://github.com/dimagi/commcare-hq/pull/25757/, so now it's 2ace5a83f6cf1259c1d11c793d6c2a2ccf9f847f

Reply:
sorry, didn't see your message about this branching off of another PR